examples/eks_cluster_webshop/helm-values/kube-prometheus-stack.yaml.tpl

```diff
@@ -0,0 +1,42 @@
+grafana:
+  persistence:
+    type: pvc
+    enabled: true
+    storageClassName: ebs-sc
+    accessModes:
+      - ReadWriteOnce
+    size: 5Gi
+    finalizers:
+      - kubernetes.io/pvc-protection
+  additionalDataSources:
+   - name: loki
+     access: proxy
+     basicAuth: false
+     editable: false
+     jsonData:
+         tlsSkipVerify: true
+     type: loki
+     url: http://loki-read-headless:3100
+  ingress:
+      ingressClassName: nginx
+      # Use cert-manager for HTTPS
+      # annotations:
+      #  cert-manager.io/cluster-issuer: selfsigned
+      enabled: true
```

Comment:
I think it's better to keep grafana private without ingress. User can use kubectl proxy to access it locally. Ideally you would expose grafana with IAM auth proxy.

Reply:
Grafana is now not expoed via ingress